chain/network/src/peer_manager/network_state/mod.rs

```diff
@@ -89,13 +90,17 @@ pub(crate) struct NetworkState {
     pub accounts_data: Arc<accounts_data::Cache>,
     /// Connected peers (inbound and outbound) with their full peer information.
     pub tier2: connection::Pool,
+    pub tier1: connection::Pool,
     /// Semaphore limiting inflight inbound handshakes.
     pub inbound_handshake_permits: Arc<tokio::sync::Semaphore>,
     /// Peer store that provides read/write access to peers.
     pub peer_store: peer_store::PeerStore,
     /// A graph of the whole NEAR network.
     pub graph: Arc<crate::routing::Graph>,
 
+    /// Hash of messages that requires routing back to respective previous hop.
```

Comment:
mention where the tier2 routing is (do you plan to move it here?)

Reply:
done